packages/block-editor/src/components/block-settings-menu-controls/index.js

```diff
@@ -25,7 +25,12 @@ import { store as blockEditorStore } from '../../store';
 
 const { Fill, Slot } = createSlotFill( 'BlockSettingsMenuControls' );
 
-const BlockSettingsMenuControlsSlot = ( { fillProps, clientIds = null } ) => {
+const BlockSettingsMenuControlsSlot = ( {
+	fillProps,
+	clientIds = null,
+	selectedClientId,
+	context,
```

Comment:
Context is required because at the moment it's impossible to determine whether the menu is triggered from list view or anywhere else. By making this a sort of `enum` type we can specify custom context. I'm thinking known future uses for this are:

- List View in the Navigation Sidebar
- List View in the Nav Block Inspector Controls.